runtimebp/cpu.go

```diff
@@ -131,7 +131,19 @@ func defaultMaxProcsFormula(n float64) int {
 // in bound of [min, max].
 //
 // Currently the default formula is NumCPU() rounding up.
+//
+// NOTE: If the GOMAXPROCS environment variable is set,
+// this function does nothing besides printing out a log,
+// even if the environment variable is set to some bogus value
+// (in that case it will be set to number of physical CPUs).
 func GOMAXPROCS(min, max int) (oldVal, newVal int) {
+	if v, ok := os.LookupEnv("GOMAXPROCS"); ok {
```

Comment:
🔕 I could be off base, but I feel like _this_ specific function should still override GOMAXPROCS and this check should maybe be in baseplate.New before calling this?

Reply:
Good point. moved to `InitFromConfig` (that's the function called by `baseplate.New`)